pg2kinesis/stream.py

```diff
@@ -41,7 +41,7 @@ def put_message(self, fmt_msg):
             self._send_agg_record(agg_record)
             self.last_send = time.time()
 
-        return agg_record
+        return agg_record or True
```

Comment:
This change forces `put_message()` to return true all the time.   In __main__.py, this means we flus the log every batch.   Is it the correct behaviour?

Reply:
again part of previous strategy we are no longer using. :)
removed